### DISCUSSION TITLE
Минимально необходимые iptables модули для работы TPWS на OpenWRT

### DISCUSSION THREAD

**Discussion by NewUse** [bol-van/zapret#416] [Q&A]
Коллеги, просьба подсказать список минимально необходимых пакетов для OpenWRT для работы TPWS через DNAT в iptables, я беглым взглядом по скрипту инсталляции выявил:

- iptables
- iptables-mod-extra
- iptables-mod-nfqueue
- iptables-mod-filter
- iptables-mod-ipopt
- iptables-mod-conntrack-extra

Но боюсь тут закралось лишнее, а места на устройстве совсем мало, для WEB UI не хватает :(((

Для минимизации ipset не нужен, может можно какие-то модули, связанные с ним выкинуть?

**Comment by bol-van** [comment]
Ничего не надо. iptables и так есть. остальное не нужно для tpws
multiport можно заменить 2 правилами на 80 и 443

**Reply by NewUse** [reply]
> Ничего не надо. iptables и так есть. остальное не нужно для tpws multiport можно заменить 2 правилами на 80 и 443

Подскажи, плз,  а установка gnu gzip   в данном случае сможет поднять производительность или он тоже не актуален?
